inc/class-sitemaps.php

```diff
@@ -355,7 +355,7 @@ function build_root_map() {
 				$query = $wpdb->prepare( "SELECT COUNT(ID) FROM $wpdb->posts {$join_filter} WHERE post_status IN ('publish','inherit') AND post_password = '' AND post_date != '0000-00-00 00:00:00' AND post_type = %s " . $where_filter, $post_type );
 
 				$count = $wpdb->get_var( $query );
-				if ( $count == 0  ) {
+				if ( $count == 0 ) {
```

Comment:
I think if `$count` is set to null, which happens when `get_var` fails, you want to enter this if statement and `continue`. 
`null == 0 === true`
`null === 0 === false`